cmake/modules/openmp.cmake

```diff
@@ -0,0 +1,10 @@
+option(USE_OpenMP "Use OpenMP" ON)
```

Comment:
Why enable OpenMP as default? I would do the opposite (noticed as I don't have openMP).

Reply:
Because CPU performance without parallelisation makes no sense. :)

You should install OpenMP.

Reply:
I agree, but people can just want to clone the repo and look at the compiler, I don't think they should install OpenMP for this.

Reply:
I see this in reverse: People that work on compiler can set the openMP option, while people that are curios usually don't, and loose interest soon. In any case, let's at least add this requirement in the readme.

Reply:
Still I would mention the requirement in the readme.

Reply:
Done!

Reply:
Thank you.